projects/kubernetes/e2e/e2e.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# cleanup resources created by previous runs
+kubectl get namespaces \
+  --output="jsonpath={range .items[?(.status.phase == \"Active\")]}{.metadata.name}{\"\n\"}{end}" \
+  | grep '^e2e-.*' \
+  | xargs -r kubectl delete namespaces
+
+skip="${GINKOGO_SKIP:-'\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]|\[HPA\]|Dashboard|Services.*functioning.*NodePort|.*NFS.*|.*Volume.*|\[sig-storage\]|.*StatefulSet.*|should\ proxy\ to\ cadvisor\ using\ proxy\ subresource'}"
+nodes="${GINKOGO_NODES:-"4"}" \
+flakeAttempts="${GINKOGO_FLAKE_ATTEMPTS:-"2"}" \
```

Comment:
A bunch of the uses of `\` in this file seem odd to me, why are they needed? (I mean the ones on `nodes=`, `flakeAttempts=`, `provide=` and `host=`, I understand the ones for the cleanup and exec bits).

Also the quoting in constructs such as `"${GINKOGO_FLAKE_ATTEMPTS:-"2"}"` is a bit odd (the effect is that `2` is unquoted I think). I'd suggest dropping the inner quotes.